docs/source/topics/packaging.rst

```diff
@@ -94,54 +95,82 @@ This directory structure is then zipped up and sent to AWS Lambda during the
 deployment process.
 
 
-Psycopg2 Example
-----------------
+Cryptography Example
+--------------------
+
+Below shows an example of how to use the
+`cryptography <https://pypi.python.org/pypi/cryptography>`__ package in a
+Chalice app for the ``python3.6`` lambda environment.
+
+We're going to leverage the ``vendor/`` directory in order to use this package
```

Comment:
That would be fine as well. I just want to make sure a user sees this and does not immediately start trying to follow the steps when it actually would have worked for them to start.